### ISSUE TITLE
does nft.stoprage still support to be used as a remote pinning service?

### ISSUE THREAD

**Issue by Nousnouss** [nftstorage/nft.storage#1191]
I've pinned a directory from my local node n configuring nft.storage as a remote pinning service using "https://api.nft.storage" as and endpoint , and my api key ,(tried it with both ips-desktop, and cli ) the directory shows in "my files" section on ipfs.storage dashbord , but stay Queued forver, the info shows that it can take up to 48 hours, but it's been a week and still stucked.
am i doing something wrong?

**Comment by dchoi27** [comment]
Yup, it is still supported. Queued pins should get cleared out in the coming days (working through a backlog from #1172), but as long as the content you were looking to add was available to the network, it should be also available in NFT.Storage (i.e., as long as the content size showing up in NFT.Storage is right). I wouldn't read too much into the "queued" label.

Closing for now, but feel free to reopen if this doesn't resolve itself later on.

**Comment by Nousnouss** [comment]
it's been  2 weeks now , and it finally failed after a while. now i'm trying to pin it once again , it failed directly after few seconds. does the 100Mb upload limit applies when using nft.storage as remote pinning service? Actually the folder i'm trying to pin is bigger than 100 MB. any suggestion on how i could fix this constant failure?

**Comment by dchoi27** [comment]
Reopened this, we'll follow up soon!

**Comment by dchoi27** [comment]
Hey, so issues like this with the Pinning Service API are really hard to debug since a lot of things are specific to individual users (the network you're on, how it handles routing, security, your local computer, etc.).

What might be helpful information is how large the directory you're trying to pin is (and whether you've tried various sizes), how that directory was created (did you just create one locally, did you create it through IPFS, etc.), and the sizes of the files you've successfully pinned.

**Comment by Nousnouss** [comment]
The directory I'm trying to pin is pretty big 17GB,  I uploaded it to Ipfs as a directory on my local node witch is 24/24 on line, 90 to 200 average peers are discovered.

**Comment by dchoi27** [comment]
Can you provide the other relevant details flagged in my previous comment (like how the directory was created, the size/type of files you were able to pin, etc.)? As I mentioned, there's no way for us to guarantee that your directory will get pinned because there's so much out of our control (your local network setup/settings, the IPFS network, etc.), and it just gets harder the larger the file is.

**Comment by Nousnouss** [comment]
here is the first CID:
ipfs://QmNbHqYeMy3rJdbwi1KZKRC77kdMRguci71kKAomkcmBa7
I also compressed the files in case it was the size that made it fails , so i came up with a little smaller version:
ipfs://QmVHsABcisVfQj9Dqc2gLNCkUtGQ3rvdZBwd6FABAifgqk still not pinning 
an then tried with very small directory:
ipfs://QmVHsABcisVfQj9Dqc2gLNCkUtGQ3rvdZBwd6FABAifgqk
That also failed to be pinned

**Comment by alanshaw** [comment]
3 of our cluster peers have tried 29 times to pin this content but for some reason it cannot be fully pinned:

```
PIN_ERROR: context canceled | 2022-02-28T14:44:49.085687764Z | Attempts: 29 | Priority: false
```

Trying locally, I only seem to be able to obtain ~20% of the directory:

```console
$ ipfs dht findprovs QmNbHqYeMy3rJdbwi1KZKRC77kdMRguci71kKAomkcmBa7
12D3KooWLsSWaRsoCejZ6RMsGqdftpKbohczNqs3jvNfPgRwrMp2
12D3KooWHXKaRAKgQbPNqgpJwojmcHUajSFnQvHdKjPRbVHRhobC
^C
$ ipfs get QmNbHqYeMy3rJdbwi1KZKRC77kdMRguci71kKAomkcmBa7
Saving file(s) to QmNbHqYeMy3rJdbwi1KZKRC77kdMRguci71kKAomkcmBa7
 1.54 GiB / 7.48 GiB [======================>----------------------------------------------------------------------------------------]  20.64% 25m07s
```

We need a peer online that has all of the content or you'll need to upload it to NFT.Storage directly.

**Comment by Nousnouss** [comment]
Strange that the whole directory can't be reached. my node is supposed to be online 7/24. Maybe some network interruption made it off.
I'm restarting daemon now , hope this will solve the problem.

**Comment by alanshaw** [comment]
Our cluster is still having trouble fetching this data from your peer.

I can connect now to `12D3KooWSqZPFHbBB5sg1M4vu4LPSY7SEvp55pazXMJFNTJty9Aw`. Are you sure this peer has ALL the data? Can you run `ipfs files stat --with-local /ipfs/QmNbHqYeMy3rJdbwi1KZKRC77kdMRguci71kKAomkcmBa7` and verify it has 100% of the data locally?
